### PR TITLE
Pod purge command should purge volumes as part of work

### PR DESCRIPTION
**Pull request by giggsoff**
Seems purge operation expect to have new volumes to recreate them

### CODE REVIEW

File: cmd/edenPod.go [hunk 1]
```diff
@@ -302,6 +305,39 @@ var podPurgeCmd = &cobra.Command{
 					app.Purge = &config.InstanceOpsCmd{Counter: 0}
 				}
 				app.Purge.Counter++
+				volumeConfigs := dev.GetVolumes()
+				for i, oldUUID := range volumeConfigs {
+					v, err := ctrl.GetVolume(oldUUID)
+					if err != nil {
+						log.Fatal(err)
+					}
+					if explicitVolumes {
```
Comment (eriknordmark) [inline cmd/edenPod.go:314]: Are either all or none of the volumes for an app instance explicit/persistent?

File: cmd/edenPod.go [hunk 2]
```diff
@@ -302,6 +305,39 @@ var podPurgeCmd = &cobra.Command{
...
+							continue
+						}
+					}
+					newUUID, err := uuid.NewV4()
+					if err != nil {
+						log.Fatal(err)
+					}
+					//update uuid to fire purge
+					v.Uuid = newUUID.String()
+					volumeConfigs[i] = newUUID.String()
```
Comment (eriknordmark) [inline cmd/edenPod.go:332]: I think the purgeCmd counter increasing was sufficient to cause the purge. Has something changed in how EVE behaves to also require a new UUID and/or generation counter for the volume+volumeref?

File: cmd/edenPod.go (hunk 1 above)
Comment (giggsoff) [reply cmd/edenPod.go:314]: If we not define volume option, I purge all, if define, purge only defined. I put comment about rhis into the cobra option comment

**Review by rvs — APPROVED** [review]
LGTM

**Pull request merged**